srcpkgs/lms/files/lms/run

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+[ -r conf ] && . ./conf
+
+: ${USER:=_lms}
+: ${CONF_FILE:=/etc/lms.conf}
+: ${WORKDIR:=/var/lib/lms}
+
+cd "$WORKDIR"
+
+exec chpst -u ${USER} lms ${CONF_FILE}
```

Comment:
I would suggest to use this:
```shell
#!/bin/sh

[ -r conf ] && . ./conf
exec 2>&1
exec chpst -u "${LMSUSER:-_lms}" -C "${WORKDIR:-/var/lms}" lms ${OPTS}
```

Reply:
Just did what you said here, pretty much agree :)